lambdanotes/src/components/editnote/EditNote.js

```diff
@@ -15,12 +20,16 @@ class EditNote extends React.Component {
     this.onFormChange = this.onFormChange.bind(this);
   }
   onFormChange = event => {
-  this.setState({[event.target.name]: event.target.value})
+    var index = this.props.match.params.id;
+    var whatIsChanging = event.target.name;
+    const editedNote = this.state.notes;
+    editedNote[index][whatIsChanging] = event.target.value;
+    this.setState({editedNote});
   }
   onUpdateClick = () => {
     this.props.editNote(this.state.notes);
   }
-  componentDidMount(){
+  componentWillMount(){
```

Comment:
Hey @Raymondgrc, `componentWillMount()` is being deprecated in React. Try switching back `componentDidMount()` 👍